pkg/storage/gcp/saver.go

```diff
@@ -12,31 +12,42 @@ import (
 	"google.golang.org/api/option"
 )
 
-// Storage implements the (github.com/gomods/pkg/storage).Saver interface
+// Storage implements the Saver interface
+// https://github.com/gomods/athens/blob/master/pkg/storage/saver.go
```

Comment:
Instead of linking to the storage saver interface in code, can you spell out the package instead? In this case, it would be `(./pkg/storage).Saver`. This notation tends to "last" over time a little better because the GH filename could change, etc...